src/py42/sdk/queries/fileevents/filters/risk_filter.py

```diff
@@ -77,7 +77,7 @@ def choices():
             RiskIndicator.CloudDataExposures.choices()
             + RiskIndicator.CloudStorageUploads.choices()
             + RiskIndicator.CodeRepositoryUploads.choices()
-            + RiskIndicator.EmaiServiceUploads.choices()
+            + RiskIndicator.EmailServiceUploads.choices()
```

Comment:
We need to make sure to get this in the next release!  These `RiskIndicator` enums thankfully have not been released yet.

Reply:
This was my doing, I'll open a separate fix for it 😞

Reply:
Thanks @alanag13  No worries :)